packages/artifact/src/internal/upload/upload-zip-specification.ts

```diff
@@ -13,6 +13,12 @@ export interface UploadZipSpecification {
    * The destination path in a zip for a file
    */
   destinationPath: string
+
+  /**
+   * Information about the file
+   * https://nodejs.org/api/fs.html#class-fsstats
+   */
+  stats: fs.Stats
```

Comment:
Since this is in the `src/internal` directory, does that mean this interface won't be used by consumers of this package?

Would it make sense to export the minimal information we need rather than the entire `fs.Stats` interface?

Reply:
The reason why I added the `fs.Stats` interface is if we eventually want to add file permissions, see this PR: 
- https://github.com/actions/toolkit/pull/1723

The stats and mode can eventually be passed to the archiver library. I didn't want to do that _yet_ to avoid additional changes.